js/viz/docs/docObjectsDxSankey.js

```diff
@@ -0,0 +1,103 @@
+
+/**
+* @name dxSankeyNodeItem
+* @publicName NodeItem
+*/
+var NodeItem = {
+    /**
+    * @name dxSankeyNodeItemfields_title
+    * @publicName title
+    * @type string
+    */
+    title: undefined,
+    /**
+    * @name dxSankeyNodeItemfields_linksIn
+    * @publicName linksIn
+    * @type Array<Object>
+    */
+    linksIn: undefined,
+    /**
+    * @name dxSankeyNodeItemfields_linksOut
+    * @publicName linksOut
+    * @type Array<Object>
+    */
+    linksOut: undefined,
+    /**
+    * @name dxSankeyNodeItemmethods_hover
+    * @publicName hover(state)
+    * @param1 state:boolean
+    */
+    hover: function () { },
+    /**
+    * @name dxSankeyNodeItemmethods_ishovered
+    * @publicName isHovered()
+    * @return boolean
+    */
+    isHovered: function () { },
+    /**
+    * @name dxSankeyNodeItemmethods_showtooltip
+    * @publicName showTooltip()
+    */
+    showTooltip: function(){ }
+};
+
+/**
+* @name dxSankeyLinkItem
+* @publicName LinkItem
+*/
+var LinkItem = {
+    /**
+    * @name dxSankeyLinkItemfields_connection
+    * @publicName connection
+    * @type Object
+    */
+    connection: undefined,
+    /**
+    * @name dxSankeyLinkItemmethods_hover
+    * @publicName hover(state)
+    * @param1 state:boolean
+    */
+    hover: function () { },
+    /**
+    * @name dxSankeyLinkItemmethods_adjacentnodehover
```

Comment:
Please, change delimiters in doc-comments from "_" to "." and use camelCase in names dxSankeyLinkItemMethods.adjacentNodeHover